README.md

```diff
@@ -36,7 +38,9 @@ SSSDK.shared.logout()
 ```
 
 ### refershAccessToken
-Refreshes the `access_token`, then updates the keychain. It also uses Salesforce's introspection endpoint to fetch and store the `expiry` of the `access_token` and saves it to keychain.
+Refreshes the `access_token`, then updates the keychain.
+(for more information please visit the following link -> https://datatracker.ietf.org/doc/html/rfc6749#section-6) 
```

Comment:
I'm not sure if this adds much value (given that we have created the SDK to abstract this from the user)

Might be place these docs in the code base for some who is looking to see how this works.